mm2src/coins/hd_wallet/address_ops.rs

```diff
@@ -1,15 +1,19 @@
 use bip32::DerivationPath;
-use std::fmt::Display;
 use std::hash::Hash;
 
+/// A trait for converting an address into a string suitable for display in logs, errors, or messages.
+pub trait DisplayAddress {
+    fn display_address(&self) -> String;
+}
+
```

Comment:
Moved this to HD wallet traits so that HD wallet mod doesn't depend on `coins` which in turn depends on HD wallet, this is because I plan to separate HD wallet to a separate crate in the future. It also makes more sense this way, as display address is a feature of a wallet not a coin.